lib/http/base.js

```diff
@@ -51,8 +52,12 @@ HttpClient.prototype.request = function (options)
   var body;
   var request;
 
-  // normalize the headers
-  headers = normalizeHeaders(options.headers);
+  headers = {
+    ...normalizeHeaders(options.headers),
+    // axios does not know how to unzip response without content-length and  https://github.com/axios/axios/issues/5298
+    // and google api returns such gzipped response https://github.com/apache/libcloud/issues/1544 and https://issuetracker.google.com/issues/177896087
+    'Accept-Encoding': options.url.includes('storage.googleapis.com') ? 'application/json' : 'gzip'
```

Comment:
Are you sure? This is something like mixing two worlds - a world of response content type and response compression...

Reply:
it was an workaround - instead of application/json it could be any string, but we don't need it anymore